src/current/molt/molt-fetch.md

```diff
@@ -369,10 +373,110 @@ To cancel replication, enter `ctrl-c` to issue a `SIGTERM` signal. This returns
 `import-only` instructs MOLT Fetch to load the source data in the specified [cloud storage](#cloud-storage) or [local file server](#local-file-server) into the CockroachDB target.
 
 {% include_cached copy-clipboard.html %}
-~~~ 
+~~~ shell
 --mode import-only
 ~~~
 
+#### Fail back to source database
+
+{{site.data.alerts.callout_danger}}
+Before using `failback` mode, refer to the [technical advisory]({% link advisories/a123371.md %}) about a bug that affects changefeeds on certain CockroachDB versions.
```

Comment:
@taroface I think we want to be more explicit here and mention versions that they are recommended to run failback with. They are the versions that are specified here: https://cockroachlabs.atlassian.net/browse/CC-30305

This is not going to change, so it is find to just document this inline so they can refer to the docs if they want more details, but we tell them up front which versions they should use.

Reply:
OK, added the versions.